README.rst

```diff
@@ -83,6 +82,62 @@ After running `aws-mfa`, your credentials file would read:
     aws_secret_access_key = <POPULATED_BY_AWS-MFA>
     aws_security_token = <POPULATED_BY_AWS-MFA>
 
+The default naming convention for the credential section can be overriden by using the ``--long-term-suffix`` and
+``--short-term-suffix`` command line arguments. For example, in a multi account scenario you can have one AWS account
```

Comment:
In a multi account scenario  you can have one AWS account in which you manage the IAM users within your organisation (recommended by Amazon). These IAM users get access to the other accounts by assuming the appropriate roles in the other accounts. That is why those users only have one set of long term credentials.